client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -22,20 +24,37 @@ const DesignSelector: FunctionComponent< Props > = () => {
 	const { selectedDesign, siteVertical } = useSelect( select =>
 		select( ONBOARD_STORE ).getState()
 	);
-	const { setSelectedDesign } = useDispatch( ONBOARD_STORE );
+	const { setSelectedDesign, resetOnboardStore } = useDispatch( ONBOARD_STORE );
+	const makePath = usePath();
+
+	const handleStartOverButtonClick = () => {
+		resetOnboardStore();
+	};
 
 	return (
 		<div
 			className="design-selector gutenboarding-color-coded"
 			data-vertical={ siteVertical?.label }
 		>
 			<div className="design-selector__header">
-				<h1 className="design-selector__title">{ NO__( 'Choose a starting design' ) }</h1>
-				<h2 className="design-selector__subtitle">
-					{ NO__(
-						'Get started with one of our top website layouts. You can always change it later'
-					) }
-				</h2>
+				<div className="design-selector__header-section design-selector__header-section--title">
+					<h1 className="design-selector__title">{ NO__( 'Choose a starting design' ) }</h1>
+					<h2 className="design-selector__subtitle">
+						{ NO__(
+							'Get started with one of our top website layouts. You can always change it later'
+						) }
+					</h2>
+				</div>
+				<div className="design-selector__header-section">
+					<Link
+						className="design-selector__start-over-button"
+						onClick={ handleStartOverButtonClick }
+						to={ makePath( Step.IntentGathering ) }
+						isLink
+					>
+						{ NO__( 'Start Over' ) }
```

Comment:
I think we should use sentence case: "Start over"

Reply:
Noted.